### PR TITLE
Consolidate pros/cons sections in Coin Selection Page

### PR DESCRIPTION
**Pull request by pavlenex**
This is an attempt to cut down and make the pros/cons sections more concise.
It doesn't restructure things, simply cuts down on text trying to simplify it.
#360

### CODE REVIEW

**Comment by pavlenex** [comment]
In https://github.com/BitcoinDesign/Guide/pull/436/commits/4ce6f2b93498bd9654175c25c16c559d04ed16c6 I removed a sentence that can potentially lead to a conclusion that the more UTXO's you select, the faster the transaction is, the only way to speed up is by paying a higher fee.

File: guide/payments/coin-selection.md [hunk 1]
```diff
@@ -165,15 +165,15 @@ Freezes certain coins or clusters from their wallet’s UTXO pool to either prio
...
 {% include fact/close.html %}
 
 {% include fact/cons.html %}
 - Doesn't optimize for all three features (speed, cost, privacy)
-- Automatic selection strategies may select UTXOs associated with targeted dusting attacks, resulting in privacy loss
-- UTXOs selected from unrecognized clusters can expose private data of wallet addresses, balances, and contact payment information (UTXO derivation paths and data).
+- Can select UTXOs associated with targeted dusting attacks, resulting in privacy loss
+- UTXOs selected from unrecognized clusters can expose private data of wallet addresses, balances, and contact payment information.
 - Reducing the inputs (byte size) of bitcoin transactions doesn’t usually optimise for minimal change outputs, exposing larger amounts of sensitive wallet data.
 - Freezing certain coins reduces the size of a wallet's UTXO pool, and may hinder an automatic algorithm’s ability to optimise for speed, cost, or privacy.
```
Comment (johnsBeharry) [inline guide/payments/coin-selection.md:178]:
```suggestion
- Freezing coins reduces the available UTXOs an automatic coin selection algorithm can choose from so the resulting transaction may not be properly optimized.
```

Comment (johnsBeharry) [reply guide/payments/coin-selection.md:178]: can be less wordy...

File: guide/payments/coin-selection.md [hunk 2]
```diff
@@ -165,17 +165,17 @@ Freezes certain coins or clusters from their wallet’s UTXO pool to either prio
...
 - Doesn't optimize for all three features (speed, cost, privacy)
-- Automatic selection strategies may select UTXOs associated with targeted dusting attacks, resulting in privacy loss
-- UTXOs selected from unrecognized clusters can expose private data of wallet addresses, balances, and contact payment information (UTXO derivation paths and data).
-- Reducing the inputs (byte size) of bitcoin transactions doesn’t usually optimise for minimal change outputs, exposing larger amounts of sensitive wallet data.
-- Freezing certain coins reduces the size of a wallet's UTXO pool, and may hinder an automatic algorithm’s ability to optimise for speed, cost, or privacy.
+- Can select UTXOs associated with targeted dusting attacks, resulting in privacy loss
+- UTXOs selected from unrecognized clusters can expose private data of wallet addresses, balances, and contact payment information.
+- Reducing the inputs doesn’t usually optimise for minimal change outputs, exposing larger amounts of sensitive wallet data.
+- Freezing coins reduces the available UTXOs an automatic coin selection algorithm can choose from so the resulting transaction may not be properly optimized.
```
Comment (johnsBeharry) [inline guide/payments/coin-selection.md:178]:
```suggestion
- Freezing UTXOs prevents coin selection algorithms from using them, so the resulting transaction may not be properly optimized.
```

File: guide/payments/coin-selection.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ Miners select transactions based on their fee, measured by the fee rate of satos
 
 #### Speed
 
-Higher fees will increase the likelihood of your transaction being prioritized by miners and included in the next block, thereby increasing the speed of your transaction. This can be optimized by either increasing the fee rate or selecting a higher number of transaction inputs.
+Higher fees will increase the likelihood of your transaction being prioritized by miners and included in the next block, thereby increasing the speed of your transaction.
```
Comment (Bosch-0) [inline guide/payments/coin-selection.md:62]:
```suggestion
Higher fees will increase the likelihood of your transaction being prioritized by miners and included in the next block, thereby increasing the speed of your transaction being confirmed.
```

Small bit but It's important to add the context that when talking about speeding up a transaction it pertains to it being confirmed in the network.

**Review by Bosch-0 — CHANGES_REQUESTED** [review]
LGTM, left a few comments

**Comment by pavlenex** [comment]
This one is good to go, needs final approval and merge, @Bosch-0 will address some points in another PR.

**Review by Bosch-0 — APPROVED** [review]

**Review by johnsBeharry — APPROVED** [review]
nice work on this!

**Comment by johnsBeharry** [comment]
Oops sorry didn't realise you wanted Christoph to review before merge. was a bit too excited to see it updated.

**Comment by pavlenex** [comment]
It's fine, either Christoph or you, first come first served 😂

**Pull request merged**